lib/internal/Magento/Framework/App/Bootstrap.php

```diff
@@ -135,7 +135,7 @@ public static function populateAutoloader($rootDir, $initParams)
     {
         $dirList = self::createFilesystemDirectoryList($rootDir, $initParams);
         $autoloadWrapper = AutoloaderRegistry::getAutoloader();
-        Populator::populateMappings($autoloadWrapper, $dirList, new ComponentRegistrar());
+        Populator::populateMappings($autoloadWrapper, $dirList);
```

Comment:
Nice! All such occurrences may be found with static code analysis, added to my TODO :)